### PR TITLE
llm: add page

### PR DESCRIPTION
**Pull request by giladAlboher**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/llm.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`llm "Ten fun names for a pet pelican"`
+
+- Run a system prompt against a file:
+
+`cat myfile.py | llm -s "Explain this code"`
+
+- Install Plugin and Local Models:
+
+`llm install llm-gpt4all`
```
Comment (kbdharun) [inline pages/common/llm.md:28]:
```suggestion
- Install packages from PyPI into the same environment as LLM:

`llm install {{package1 package 2 ...}}`
```

File: pages/common/llm.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Download and run a prompt against the Orca Mini 7B model:
+
+`llm -m orca-mini-3b-gguf2-q4_0 'What is the capital of France?'`
```
Comment (kbdharun) [inline pages/common/llm.md:32]:
```suggestion
- Download and run a prompt against a [m]odel:

`llm -m {{orca-mini-3b-gguf2-q4_0}} '{{What is the capital of France?}}'`
```

File: pages/common/llm.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Interactive Chat:
+
+`llm chat -m chatgpt`
```
Comment (kbdharun) [inline pages/common/llm.md:36]:
```suggestion
- Have an interactive chat with a specific [m]odel:

`llm chat -m {{chatgpt}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Tip**: You can batch all my suggestions from the "File changed" tab and make a single commit.

**Review by kbdharun — APPROVED** [review]

**Comment by walking-octopus** [comment]
I think `llm install` seems less like a part of core functionality than say `llm models`. Besides, should features like embeddings be documented?

File: pages/common/llm.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# LLM
```
Comment (kbdharun) [inline pages/common/llm.md:1]:
```suggestion
# llm
```
Note. The page headings must match command name.

File: pages/common/llm.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# llm
+
+> A utility and Python library for interacting with Large Language Models (LLMs) via remote APIs and models that can be installed and run on your machine.
```
Comment (acuteenvy) [inline pages/common/llm.md:3]:
```suggestion
> Interact with Large Language Models (LLMs) via remote APIs and models that can be installed and run on your machine.
```

File: pages/common/llm.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+> More information: <https://llm.datasette.io/en/stable/help.html>.
+
+- Set up OpenAI API Key:
```
Comment (acuteenvy) [inline pages/common/llm.md:6]:
```suggestion
- Set up an OpenAI API Key:
```

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
> Looking at the [documentation page](https://llm.datasette.io/en/stable/help.html), I suggest we use separate pages for the subcommands, just like we do/did for other commands.

Yep, we can convert the issue to a let's document.

File: pages/common/llm.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+`llm keys set openai`
+
+- Run a prompt:
+
+`llm "{{Ten fun names for a pet pelican}}"`
+
+- Run a [s]ystem prompt against a file:
+
+`cat {{path/to/file.py}} | llm -s "{{Explain this code}}"`
```
Comment (acuteenvy) [inline pages/common/llm.md:16]: Please use long options if available: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax
```suggestion
`cat {{path/to/file.py}} | llm --system "{{Explain this code}}"`
```

File: pages/common/llm.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+- Install packages from PyPI into the same environment as LLM:
+
+`llm install {{package1 package2 ...}}`
+
+- Download and run a prompt against a [m]odel:
+
+`llm -m {{orca-mini-3b-gguf2-q4_0}} '{{What is the capital of France?}}'`
```
Comment (acuteenvy) [inline pages/common/llm.md:24]:
```suggestion
`llm --model {{orca-mini-3b-gguf2-q4_0}} "{{What is the capital of France?}}"`
```

File: pages/common/llm.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+- Have an interactive chat with a specific [m]odel:
+
+`llm chat -m {{chatgpt}}`
```
Comment (acuteenvy) [inline pages/common/llm.md:28]:
```suggestion
`llm chat --model {{chatgpt}}`
```

**Review by acuteenvy — APPROVED** [review]

**Comment by kbdharun** [comment]
> Should i do anything else?

Nope, this PR is GTG. Thanks for your contribution as always 😄 .

**Comment by kbdharun** [comment]
> I thought we would make this page a main page for llm and refer to the subcommand pages, as described in the Let's document issue?

We can reference/update this main page once we have documented the subcommands (for now this main page is sufficient and since it isn't a blocking issue I merged this).

**Pull request merged**